modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/GridCacheFullTextQuerySelfTest.java

```diff
@@ -59,6 +59,19 @@
     /** Cache name */
     private static final String PERSON_CACHE = "Person";
 
+    /** Limitation to query response size */
+    private static final int QUERY_LIMIT = 5;
+
+    private static final class TestPair{
+
+        public final Set<Integer> expected;
+        public final List<Cache.Entry<Integer, ?>> all = new ArrayList<>();
+
+        public TestPair(Set<Integer> exp) {
+            this.expected = new HashSet<>(exp);
+        }
+    }
```

Comment:
This class also should have minimal javadoc.

https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-JavadocComments